src/Libraries/Microsoft.Extensions.Caching.Hybrid/Internal/DefaultHybridCache.Serialization.cs

```diff
@@ -0,0 +1,121 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Concurrent;
+using System.ComponentModel;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using Microsoft.Extensions.DependencyInjection;
+
+#if !(NETCOREAPP2_0_OR_GREATER || NETSTANDARD2_1_OR_GREATER)
+using System.Runtime.InteropServices;
+using System.Runtime.Serialization;
+#endif
+
+namespace Microsoft.Extensions.Caching.Hybrid.Internal;
+internal partial class DefaultHybridCache
```

Comment:
[Styling] Missing blank line.
Also, unnecessary blank lines in the usings block.

Reply:
in the usings: is separating a conditional set (TFM-dependent - otherwise get "remove unused using" on some TFM builds); IMO much clearer with the separation